Makefile

```diff
@@ -46,7 +46,7 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
-OPERATOR_SDK_VERSION=v1.26.1
+OPERATOR_SDK_VERSION ?= v1.34.1
```

Comment:
no other line below has ?=, why that changed here?

Reply:
so we can change it from outside if needed

Reply:
I suppose all the lines below should have that ?= for consistency.

Reply:
done